frontend/src/app/shared/components/file-links/file-link-list/file-link-list.html

```diff
@@ -19,6 +19,7 @@
     <li
       *ngFor="let fileLink of fileLinks$ | async; let i = index;"
       class="spot-list--item op-files-tab--file-list-item"
+      [ngClass]="{ 'disabled': fileLink._links.permission.href !== 'urn:openproject-org:api:v3:file-links:permission:View' }"
```

Comment:
Same as above. The capabilities API might be better. If you can't use it, please extract the `urn:openproject-org...` into constants at least so that we don't have to maintain multiple strings at different places.